tests/test_client.py

```diff
@@ -381,33 +376,37 @@ def test_detector_creation():
     add_mock_detector_creation_response()
     client.create_detector()
     for good_arg in good_args:
-        add_mock_detector_creation_response(**good_arg)
+        default_args = get_default_args(client.create_detector, ['name'])
```

Comment:
Seems like we are somewhat changing the logic of the test here, no ?

Reply:
Agree, not worth.